src/fontra/views/editor/scene-model.js

```diff
@@ -103,6 +103,12 @@ export class SceneModel {
     return getSelectedGlyphName(this.selectedGlyph, this.glyphLines);
   }
 
+  isSelectedGlyphLocked() {
+    return !!this.getSelectedPositionedGlyph()?.varGlyph?.glyph.customData?.[
```

Comment:
Just a tiny nit:
```suggestion
    return !!this.getSelectedPositionedGlyph()?.varGlyph?.glyph.customData.[
```
Once we have a glyph, customData is guaranteed to exist.